eos/form-factors/mesonic.cc

```diff
@@ -3,7 +3,6 @@
 /*
  * Copyright (c) 2010, 2011, 2013, 2014, 2015, 2016, 2018 Danny van Dyk
  * Copyright (c) 2015 Christoph Bobeth
- * Copyright (c) 2010, 2011 Christian Wacker
```

Comment:
Why did you removed this entry?

Reply:
Because his code was moved to a different file.

Reply:
Ah I see, good